type_description_interfaces/msg/Field.msg

```diff
@@ -0,0 +1,8 @@
+# Represents a single field in a type.
+
+# Name of the field.
+string name
+# Type of the field, including details about the type like length, nested name, etc.
+FieldType type
+# Literal default value of the field as a string.
```

Comment:
serialized how?

Reply:
I added a comment about this in the docstring, basically stating that for now (we might want to change this later) the value in the string is exactly the string used for the default value in the originating text, whether that was .msg or .idl.